packages/babel-plugin-transform-react-display-name/README.md

```diff
@@ -7,15 +7,19 @@
 **In**
 
 ```js
-var foo = React.createClass({});
+var foo = React.createClass({}); // React <= 15
```

Comment:
dono if we want to modify ` > Add displayName to React.createClass calls` above this too